static/js/components/tiles/scatter_tile.tsx

```diff
@@ -78,19 +79,20 @@ export function ScatterTile(props: ScatterTilePropType): JSX.Element {
       props.statVarSpec,
       setRawData
     );
+    setErrorMsg("");
```

Comment:
this isn't needed unless we are refetching data

Reply:
leaving this to catch the case if we ever refetch data.